modules/oe_corporate_countries_address/tests/src/Functional/CorporateCountriesFilterTest.php

```diff
@@ -0,0 +1,200 @@
+<?php
+
+namespace Drupal\Tests\oe_corporate_countries_address\Functional;
+
+use Drupal\Tests\BrowserTestBase;
+
+/**
+ * Tests for creating content type and testing the exposed country filter.
+ *
+ * @group oe_corporate_countries
+ */
+class CorporateCountriesFilterTest extends BrowserTestBase {
```

Comment:
Personally I don't think we need this whole test case, the other test case already checks what we need.